app/models/video_paper.rb

```diff
@@ -52,7 +52,7 @@ def share(shared_paper_params)
     shared_paper = SharedPaper.new(shared_paper_params)
     shared_paper.video_paper = self
     if shared_paper.save
-      ShareMailer.share_email(shared_paper.user,shared_paper.video_paper,self.user,shared_paper.notes).deliver
+      ShareMailer.share_email(shared_paper.user,shared_paper.video_paper,self.user,shared_paper.notes).deliver_now
```

Comment:
Maybe we should add a TODO so we change this to work asynchronously, so we aren't blocking on the send through AWS? I'm guessing that is what `_now` part means.

Reply:
Rails 4.2 changed `deliver` to `deliver_now` and added a `deliver_later` that uses redis or delayed jobs.

Reply:
Right, that is what I guessed. So we should probably add a TODO to decide if we should use delayed job to send the email instead of sending synchronously.